ee/control/consumers/flareconsumer/flarerunner.go

```diff
@@ -0,0 +1,15 @@
+package flareconsumer
+
+import (
+	"context"
+	"io"
+
+	"github.com/kolide/launcher/pkg/agent/types"
+	"github.com/kolide/launcher/pkg/debug/checkups"
+)
+
+type FlareRunner struct{}
```

Comment:
Why have this split out?

Reply:
Because the checkups.RunFlare is purely functional and I need to pass an interface to the consumer for testing ... but I suppose I could also just pass it a function.

Reply:
or did you mean, why have this split into its own file?

Reply:
I mean why have the struct, and you answered